scripts/exrrfs_fcst.sh

```diff
@@ -38,7 +38,8 @@ fi
 ln -snf ${UMBRELLA_DATA}/cycbdys/lbc*.nc .
 
 ln -snf ${FIXrrfs}/physics/${PHYSICS_SUITE}/* .
-ln -snf ${FIXrrfs}/meshes/3km_conus.ugwp_oro_data.nc .
+ln -snf ${FIXrrfs}/meshes/${MESH_NAME}.ugwp_oro_data.nc ./ugwp_oro_data.nc
+ln -snf ${FIXrrfs}/meshes/${MESH_NAME}.invariant.nc_L65 ./invariant.nc
```

Comment:
We need to link different `invariant.nc` based on different vertical level levels.

One possible option is we copy the given level file into the EXPDIR directory as `zeta_levels.txt` and then we get the corresponding information from it. This will simply the workflow logic.